scripts/pre_push_hook.py

```diff
@@ -43,7 +43,7 @@
 sys.path.append(os.getcwd())
 from scripts import common  # isort:skip  # pylint: disable=wrong-import-position
 from scripts import install_backend_python_libs # isort:skip  # pylint: disable=wrong-import-position
-from core import python_utils  # isort:skip  # pylint: disable=wrong-import-position
+# isort:skip  # pylint: disable=wrong-import-position
```

Comment:
Remove these unwanted bypass-flags

Reply:
@anurag629 Is this comment resolved? (Please make sure to reply to the comments once it's resolved!)

Reply:
yes